### PR TITLE
document otel.instrumentation.common.db-statement-sanitizer.enabled

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

File: content/en/docs/zero-code/java/spring-boot/_index.md [hunk 1]
```diff
@@ -431,6 +431,12 @@ Automatic instrumentation is available for several frameworks:
 | Micrometer            | `otel.instrumentation.micrometer.enabled`       | false         |
 | R2DBC (reactive JDBC) | `otel.instrumentation.r2dbc.enabled`            | true          |
 
+Common properties for all instrumentations:
```
Comment (jeanbisutti) [inline content/en/docs/zero-code/java/spring-boot/_index.md:434]:
```suggestion
Common properties for all database instrumentations:
```

**Review by theletterf — APPROVED** [review]

**Review by trask — APPROVED** [review]

**Review by jeanbisutti — APPROVED** [review]

**Review by chalin — APPROVED** [review]
Thanks all ✨

**Pull request merged**